src/browser/api/application.js

```diff
@@ -132,6 +132,7 @@ Application.create = function(opts, configUrl = '', parentIdentity = {}) {
 
     let appUrl = opts.url;
     const { uuid, name } = opts;
+    const initialAppOptions = Object.assign({}, opts);
```

Comment:
Just make sure it's okay that it is only a shallow clone

Reply:
:+1: we good